#goal definition
float32 yaw_setpoint
---
#result definition
float32 yaw_final
---
#feedback
float32 yaw_error
